### DISCUSSION TITLE
copying mods from score

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#20336] [Ideas]
if you hold shift/control/other modifier whilst clicking on a score in leaderboards, it should copy the mods.
 
eg. clicking on this enables the traceable mod. 
![image](https://user-images.githubusercontent.com/78441708/190542948-2fda2a39-1c29-4dca-97f4-ee8e2a1da985.png)

just a nice QOL feature I would appreciate :)

**Comment by smoogipoo** [comment]
You can right click the mods and select "use these mods".

**Reply by Axilotl17** [reply]
I know, but I think a shortcut would also be useful?

**Comment by zhevv** [comment]
I decided to implement shift+clicking to copy mods since I do feel the same in terms of convenience.

https://github.com/ppy/osu/assets/136142795/a6a81525-4f10-492d-9229-4c85794ce6f7

I'd also like to point out that the "Use these mods" option only works when you right click a mod icon, which I guess makes sense, but is slightly inconvenient + can't copy a nomod score. Perhaps the option can be made available no matter where you right click on the score?

**Reply by peppy** [reply]
Making it work anywhere on the score would be best, yes.

**Reply by zhevv** [reply]
Your opinion on the shift clicking? If its good then I'll go ahead with the PR. https://github.com/zhevv/osu/tree/copy-mod-from-scores

**Reply by peppy** [reply]
Please no shift click.

**Reply by okolenmi** [reply]
![изображение](https://github.com/ppy/osu/assets/55319814/19bc9236-3010-4aaf-8ec7-c3babb392fa9)
Extra opinion: it can be a button that will appear after hover event under mods. It will disappear after hover event is not active on whole score. Honestly, current "use these mods" feature is like an easter egg that very difficult to discover (I have discovered it here after 2years of playing).
